docs/openapi.json

```diff
@@ -192,7 +190,6 @@
           "Text Generation Inference"
         ],
         "summary": "Generate a stream of token using Server-Sent Events",
```

Comment:
Why are the descriptions gone ?

Reply:
this is related to an upstream change where utopia added support for a custom description instead of using the same value for both description and summary (related PRs https://github.com/juhaku/utoipa/issues/781 and https://github.com/juhaku/utoipa/pull/948).